src/components/form/Form.tsx

```diff
@@ -27,9 +28,13 @@ const Form = () => {
       });
   };
 
+  const { mutate } = useMutation({
```

Comment:
FYI, destructuring may be ok if you have only one mutation, but in larger applications, it's usually a case that you have multiple. It's better to name the mutation and then use it like `mutationName.mutate`

Reply:
oh good to know!